lib/src/fitness.dart

```diff
@@ -71,29 +76,73 @@ Future<Fitness> calcFitness(
   final warnPoints = max(4.0, magnitude * 0.05); // 5%
   final hintPoints = 1.0;
 
+  final suggestions = <Suggestion>[];
   var shortcoming = 0.0;
 
   if (platform != null && platform.hasConflict) {
     shortcoming += errorPoints;
+    final components = platform?.components?.map((s) => '`$s`')?.join(', ');
+    var description =
+        'Make sure that the imported libraries are not in conflict.';
+    if (components != null) {
+      description += ' Detected components: $components.';
+    }
+    suggestions.add(new Suggestion.error(
+      'Fix platform conflict in `$dartFile`.',
+      description,
+      file: dartFile,
+      penalty: new Penalty(fraction: 2000, amount: errorPoints.ceil()),
+    ));
   }
 
   if (isFormatted == null || !isFormatted) {
     shortcoming += hintPoints;
+    suggestions.add(new Suggestion.hint(
+      'Format `$dartFile`.',
+      'Run `dartfmt` to format `$dartFile`.',
+      file: dartFile,
+      penalty: new Penalty(amount: hintPoints.ceil()),
+    ));
+  }
+
+  String suggestionDescription(CodeProblem cp, String local) {
+    return 'Strong-mode analysis of `$dartFile` $local:\n\n'
```

Comment:
Is it always strong-mode, or does it depend on analyzer-options.yaml?

Since it'll eventually be the only mode, maybe we should just say 'Analysis of...', but we can leave that for a later change.

Reply:
I like to include it for now, because it raises awareness, but we may omit that after Dart 2 is finished. Added a TODO to change it later.